features/_your_results/your_results_calculating_different_salary_frequencies.feature

```diff
@@ -1,10 +1,25 @@
-Feature:
-  As an enrolee into a Workplace Pension,
-  I would like to calculate pension contribution changing my frequency salary.
-  So I can see my monthly, weekly, per 4 weeks contribution.
+Feature: displaying different salary frequencies
+  In order to see my contributions in format I understand,
+  As a user of the WPCC tool,
+  I want to be able to choose the frequency my salary is broken into.
```

Comment:
The wording of this is not quite right. Not sure of a better way...
`I want to be able to choose the frequency with which my salary is calculated`